apps/web-giddh/src/app/theme/giddh-datepicker/giddh-datepicker.module.ts

```
@@ -21,11 +21,27 @@ export const GIDDH_DATEPICKER_FORMAT = {
 };
 
 export class PickDateAdapter extends NativeDateAdapter {
+    parse(value: any, parseFormat: Object): Date | null {
```

Biome (1.9.4)

[error] Don't use 'Object' as a type.

Prefer explicitly define the object shape. This type means "any non-nullable value", which is slightly better than 'unknown', but it's still a broad type.

(lint/complexity/noBannedTypes)